Kinvey/Kinvey/Persistable.swift

```diff
@@ -93,6 +94,40 @@ public func <- <Transform: TransformType>(left: inout Transform.Object!, right:
     kinveyMappingType(left: right.0, right: right.1.currentKey!)
     left <- (right.1, right.2)
 }
+  
+class ListValueTransform<T: RealmSwift.Object>: TransformOf<List<T>, [JsonDictionary]> where T: BaseMappable {
```

Comment:
`TransformOf` actually implements `TransformType` plus it checks the type inside `fromJSON()`, so no big difference despite the fact that using `TransformOf` you end up writing a little less code.
Let me know if you really thinks that using `TransformType` would be better.